lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -4127,19 +4127,25 @@ LLVMToSPIRVBase::transBuiltinToInstWithoutDecoration(Op OC, CallInst *CI,
     // Literal S Literal I Literal rI Literal Q Literal O
 
     Type *ResTy = CI->getType();
+
+    auto OpItr = CI->value_op_begin();
+    auto OpEnd = OpItr + CI->getNumArgOperands();
+
+    // If return type of intrinsic is greater than 64 then FE moves return
+    // variable into arguments list
```

Comment:
```suggestion
    // If the return type of an instruction is wider than 64-bit, then this instruction will return
    // via 'sret' argument added into the arguments list. Here we reverse this, removing 'sret'
    // argument and restoring the original return type.
```

Reply:
Changed comments in [80b3711](https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/1213/commits/80b371198f9d197467986749273b905ffeb3dbff)